tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -48,6 +48,11 @@ public function tearDown(): void
         unset($this->adapter);
     }
 
+    private function usingMysql8(): bool
+    {
+        return version_compare($this->adapter->getAttribute(\PDO::ATTR_SERVER_VERSION), '8') > -1;
```

Comment:
```suggestion
        return version_compare($this->adapter->getAttribute(\PDO::ATTR_SERVER_VERSION), '8.0.0', '>=');
```

Reply:
Did not know about that third parameter, done.